static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -15,13 +17,29 @@ export default class FilterVisibilityToggle extends React.Component {
     return checkFilterVisibility(filterName) ? "" : "closed";
   };
 
-  openStateIcon: Function = (): React$Element<*> => (
-    <Icon
+  isInResults: Function = (id: string): boolean => {
+    if (this.getResults()) {
+      const elmId = (id === "birth_location") ? "profile.birth_country3" : id;
+      const docCount = _.get(this.getResults(), ['aggregations', elmId, 'doc_count']) || 0;
```

Comment:
`_.get`'s third parameter is the value returned when it can't find the value at the path given. Can you use that instead of `|| 0`?